english/yoga/mind/yoga_mind.md

```diff
@@ -3,3 +3,5 @@ a feeling of oneness with the universe... you are one with the mushroom!
 Oops! I mean your soul.
 
 Want to know about Yoga lifestyle? Click [here](../lifestyle/yoga_lifestyle.md)
+
+Most people have discovered that exercising, whether it’s going for a walk or doing a vigorous yoga class, can lift their mood.
```

Comment:
Hi 
I tried to merge Udacity / master with my master and I couldn't . I'm not sure why. Can you please write me the reason.
Thanks,
Vered



> On Sep 2, 2015, at 9:55 AM, Casey Collaborator <notifications@github.com> wrote:
> 
> In english/yoga/mind/yoga_mind.md:
> 
> > @@ -3,3 +3,5 @@ a feeling of oneness with the universe... you are one with the mushroom!
> >  Oops! I mean your soul.
> >  
> >  Want to know about Yoga lifestyle? Click [here](../lifestyle/yoga_lifestyle.md)
> > +
> > +Most people have discovered that exercising, whether it’s going for a walk or doing a vigorous yoga class, can lift their mood.
> This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line. No need to add blank lines in between, just press enter somewhere in the middle of the line.
> 
> —
> Reply to this email directly or view it on GitHub.
>